🐛 Bug Report: Inaccurate Instructions in README.md

### What happened?

The `README.md` says to run `npm run start` in the root folder.
There is no `start` script in the root folder.

![image](https://user-images.githubusercontent.com/36056652/185731364-12687ac5-47b9-4cf5-8e42-ca30cd713914.png)

Executing `npm run start` in the root folder shows the following error.

```
npm ERR! Missing script: "start"
npm ERR!
npm ERR! Did you mean one of these?
npm ERR!     npm star # Mark your favorite packages
npm ERR!     npm stars # View packages marked as favorites
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\[User]\AppData\Local\npm-cache\_logs\2022-08-20T06_02_33_475Z-debug-0.log
```


### What you expected to happen

There is no need to execute `npm run start` in the root folder.

### How to reproduce

1. Clone the repository
2. Run `npm run start` in the root folder.

### Amplication version

_No response_

### Environment

_No response_

### Are you willing to submit PR?

Yes I am willing to submit a PR!

--- a/README.md
+++ b/README.md
@@ -84,7 +84,6 @@ Amplication is using a mono-repo with multiple packages. To initialize all the p
 ```jsx
 npm install
 npm run setup:dev
-npm run start
 ```
 This will install all the required dependencies, run the necessary scripts and init a Docker-based Postgres server.
 

